README.md

```diff
@@ -695,6 +705,8 @@ attacks. Don't do that* unless you have good reason to trust their origin.
 sanitize-html will log a warning if these tags are allowed, which can be
 disabled with the `allowVulnerableTags: true` option.
 
+### Discarding the entire contents of a disallowed tag
```

Comment:
"Choose what to do with disallowed tags" could have three fourth-level subheadings now, organizing the two existing options plus your new one.

Reply:
I added subheadings for 'discard', 'completelyDiscard' and  'recursiveEscape'